lib/stomp_client.ex

```diff
@@ -394,8 +394,8 @@ defmodule StompClient do
           end
         end
 
-      x ->
-        x
+      :partial ->
+        {:partial, message}
     end
```

Comment:
ok, then instead of `x -> x` this code should be:
`
:partial ->
  {:partial, message}
{:error, remain} ->
  {:error, remain}
`